lib/commonConfig.js

```diff
@@ -235,6 +236,11 @@ function processEnvironmentVariables() {
         config.defaultResource = process.env.IOTA_DEFAULT_RESOURCE;
     }
 
+    // Default explicitAttrs
+    if (process.env.IOTA_DEFAULT_EXPLICITATTRS !== undefined) {
```

Comment:
A couple of comments:

1. The flag (and associated env var) should be documented in installagionguide.md. Have a look to `appendMode` and `timestamp` documented there and do it in a similar way. Don't forget to include the last sentece: "This flag is overwritten by ... flag in group or device provision."
2. Change the name of the configuration parameter `defaultExplicitAttrs` -> `explicitAttrs` (consecuently, using IOTA_EXPLICT_ATTRS as env var). This way, the name of the configuraiton parameter will match with the one used in the JSON payload at group or device provisining.

Reply:
> 1. The flag (and associated env var) should be documented in installagionguide.md. Have a look to appendMode and timestamp documented there and do it in a similar way. Don't forget to include the last sentece: "This flag is overwritten by ... flag in group or device provision."

Fixed in ac1120b

> 2. Change the name of the configuration parameter defaultExplicitAttrs -> explicitAttrs (consecuently, using IOTA_EXPLICT_ATTRS as env var). This way, the name of the configuraiton parameter will match with the one used in the JSON payload at group or device provisining.

Fixed in 051c7d9